parseAndPopulate/populate.py

```diff
@@ -88,7 +88,7 @@ def __init__(self):
                 'flag': '--sdo',
                 'help': 'If we are processing sdo or vendor yang modules',
                 'action': 'store_true',
-                'default': False
+                'default': True
```

Comment:
This should stay as is. With `'action': 'store_true'` and the default being True, there would be no way to specify False.

Reply:
The main problem is that our default configuration is that `sdo` is False but default `dir` is `/var/yang/nonietf/yangmodels/yang/standard/ietf/RFC`. However, this directory is SDO. If someone runs `populate.py` script with default configuration it results in an error.

Reply:
In that case, we should either remove the `--sdo` flag and replace it with a `--vendor` flag, or add a `--vendor` flag and make passing one of the two required. Otherwise there will be no way to populate vendor directories.

Reply:
so can I just change default ```--dir``` to ```/var/yang/nonietf/yangmodels/yang/standard/iana``` and keep default ```--sdo``` as ```False``` then?

Reply:
`/var/yang/nonietf/yangmodels/yang/standard/iana` is also SDO. Vendor directories are for example
- /var/yang/nonietf/yangmodels/yang/vendor/cisco
- /var/yang/nonietf/yangmodels/yang/vendor/huawei
- /var/yang/nonietf/yangmodels/yang/vendor/fujitsu

I think you can change `--dir` to the `/var/yang/nonietf/yangmodels/yang/vendor/huawei/network-router/8.20.0/atn980b`

Reply:
thanks, done